src/reducers/linter.tsx

```diff
@@ -220,6 +220,10 @@ export const fetchLinterMessagesIfNeeded = ({
       fileId: version.fileId,
     });
 
+    if (!response) {
```

Comment:
Is this safe to add? It feels dangerous to be changing production code in order to fix a problem in a test.

Reply:
I'm not entirely sure it only happens in tests, it's hard to know without digging deeper into the code. Dispatching an action here is clearly wrong - the underlying action code does not handle being passed those falsey `response` objects.

It was already bad before, it's just more apparent with Node 16: Node 14 used to just warn, Node 16 terminates the process because we end up with an exception thrown and not caught in an async function.

Reply:
But is *this* the fix? Again, it feels dangerous to me to just `return` and act as if everything is ok.